harness/determined/keras/_tf_keras_trial.py

```diff
@@ -9,6 +9,7 @@
 from abc import abstractmethod
 from typing import Any, Dict, List, Optional, cast
 
+import cloudpickle
```

Comment:
what's wrong with `pickle`?

Reply:
Nothing - we don't need the extra functionality here, just saw we used cloudpickle for PyTorch models and thought I'd be consistent. I see we're using regular pickle elsewhere anyway though, so I can change it back.